### ISSUE TITLE
commit cb3098e makes Olive to crash on startup on macOS

### ISSUE THREAD

**Issue by pgilfernandez** [olive-editor/olive#367]
Previous one runs nicely

**Comment by itsmattkc** [comment]
Please provide a crash log, it starts up fine on macOS for me

**Comment by itsmattkc** [comment]
When it says Olive quit unexpectedly, there's a button for "Show Details" that will give you a crash log

**Comment by ion12** [comment]
For me it doesn't crash on startup either, but it does crash immediately if I try to open the 'Tools' menu.

```
Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000009
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Segmentation fault: 11
Termination Reason:    Namespace SIGNAL, Code 0xb
Terminating Process:   exc handler [0]

VM Regions Near 0x9:
-->
    __TEXT                 000000010aeac000-000000010afbd000 [ 1092K] r-x/rwx SM=COW  /Users/USER/*/Olive.app/Contents/MacOS/Olive

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   org.qt-project.QtWidgets      	0x000000010d6cb879 QAction::data() const + 9
1   ion12.Olive                   	0x000000010aec099a MainWindow::set_bool_action_checked(QAction*) + 26
2   ion12.Olive                   	0x000000010aec1dde MainWindow::toolMenu_About_To_Be_Shown() + 286
3   org.qt-project.QtCore         	0x000000010cdbf79a QMetaObject::activate(QObject*, int, int, void**) + 2954
4   org.qt-project.QtWidgets      	0x000000010d84eada QMenuPrivate::_q_platformMenuAboutToShow() + 410
5   org.qt-project.QtCore         	0x000000010cdbf79a QMetaObject::activate(QObject*, int, int, void**) + 2954
6   libqcocoa.dylib               	0x0000000110a99e65 -[QCocoaNSMenuDelegate menuWillOpen:] + 133
7   com.apple.AppKit              	0x00007fff7d1f4ebe -[NSMenu _sendMenuOpeningNotification:] + 106
8   com.apple.AppKit              	0x00007fff7cf4ed0a -[NSCarbonMenuImpl _carbonOpenEvent:handlerCallRef:] + 93
9   com.apple.AppKit              	0x00007fff7cd4e5eb NSSLMMenuEventHandler + 1093
10  com.apple.HIToolbox           	0x00007fff7e600d85 DispatchEventToHandlers(EventTargetRec*, OpaqueEventRef*, HandlerCallRec*) + 1708
11  com.apple.HIToolbox           	0x00007fff7e5ffff6 SendEventToEventTargetInternal(OpaqueEventRef*, OpaqueEventTargetRef*, HandlerCallRec*) + 428
12  com.apple.HIToolbox           	0x00007fff7e5ffe3f SendEventToEventTargetWithOptions + 43
13  com.apple.HIToolbox           	0x00007fff7e6748a6 SendMenuOpening(MenuSelectData*, MenuData*, double, unsigned int, unsigned int, __CFDictionary*, unsigned char, unsigned char*) + 484
14  com.apple.HIToolbox           	0x00007fff7e68ea57 DrawTheMenu(MenuSelectData*, __CFArray**, unsigned char, unsigned char*) + 280
15  com.apple.HIToolbox           	0x00007fff7e68e839 MenuChanged(MenuSelectData*, unsigned char, unsigned char) + 541
16  com.apple.HIToolbox           	0x00007fff7e7a9bae TrackMenuCommon(MenuSelectData&, unsigned char*, SelectionData*, MenuResult*, MenuResult*) + 1130
17  com.apple.HIToolbox           	0x00007fff7e68e223 MenuSelectCore(MenuData*, Point, double, unsigned int, OpaqueMenuRef**, unsigned short*) + 554
18  com.apple.HIToolbox           	0x00007fff7e68df66 _HandleMenuSelection2 + 460
19  com.apple.AppKit              	0x00007fff7cd3e368 _NSHandleCarbonMenuEvent + 239
20  com.apple.AppKit              	0x00007fff7cfb8702 _DPSEventHandledByCarbon + 54
21  com.apple.AppKit              	0x00007fff7d3390c5 -[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 963
22  com.apple.AppKit              	0x00007fff7cbb23db -[NSApplication run] + 926
23  libqcocoa.dylib               	0x0000000110a95a8d QCocoaEventDispatcher::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) + 2397
24  org.qt-project.QtCore         	0x000000010cd8a0a2 QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) + 418
25  org.qt-project.QtCore         	0x000000010cd8ec62 QCoreApplication::exec() + 402
26  ion12.Olive                   	0x000000010aeb28e5 main + 1029
27  libdyld.dylib                 	0x00007fff94c9b235 start + 1

Thread 0 crashed with X86 Thread State (64-bit):
  rax: 0x864dfd6622190018  rbx: 0x00007fff54d524d0  rcx: 0x0000000000000000  rdx: 0x0000000000000004
  rdi: 0x00007fff54d524d0  rsi: 0x0000000000000001  rbp: 0x00007fff54d524b0  rsp: 0x00007fff54d524a0
   r8: 0x0000000000000004   r9: 0x00007fff54d51dfc  r10: 0x00007fff9daaa101  r11: 0x00007ff4ef5089d0
  r12: 0x000000000000002e  r13: 0x000000010d0cabf0  r14: 0x0000000000000001  r15: 0x00007fff54d539e0
  rip: 0x000000010d6cb879  rfl: 0x0000000000010206  cr2: 0x0000000000000009

Logical CPU:     2
Error Code:      0x00000004
Trap Number:     14
```

**Comment by ion12** [comment]
yup, sorry I see this has already been addressed in #369 
thanks!

**Comment by itsmattkc** [comment]
@pgilfernandez Hmm I'm guessing your crash was probably related to @ion12's, if not and it comes back feel free to re-open
